CrashReporter.xcodeproj/xcshareddata/xcschemes/Disk Image.xcscheme

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we really use this target/scheme?

Reply:
Yes! Actually it's the main one for release